tools/accuracy_checker/configs/open-closed-eye-0001.yml

```diff
@@ -0,0 +1,44 @@
+models:
+
+  - name: open-closed-eye-0001
+    launchers:
+      - framework: onnx_runtime
+        model: public/open-closed-eye-0001/open-closed-eye-0001.onnx
+        adapter: classification
+
+    datasets:
+      - name: mrlEyes_2018_01
+        preprocessing:
+          - type: resize
+            size: 32
+          - type: normalization
+            mean: 128.0, 128.0, 128.0
+            std: 256.0, 256.0, 256.0
+
+  - name: open-closed-eye-0001
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   public/open-closed-eye-0001/FP32/open-closed-eye-0001.xml
+        weights: public/open-closed-eye-0001/FP32/open-closed-eye-0001.bin
+        adapter: classification
+
+      - framework: dlsdk
+        tags:
+          - FP16
+        model:   public/open-closed-eye-0001/FP16/open-closed-eye-0001.xml
+        weights: public/open-closed-eye-0001/FP16/open-closed-eye-0001.bin
+        adapter: classification
+
+    datasets:
+      - name: mrlEyes_2018_01
+        preprocessing:
+          - type: resize
+            size: 32
+            aspect_ratio_scale: greater
```

Comment:
why for onnx do you use fixed size resize policy and aspect ratio preserving for IE?
Also note that it leads to different input size after preprocessing.  You need to handle it (cropping, padding or dynamic input shape shlould be used)

Reply:
???? My input is image 32x32. And i want resize to it. Katya, what i have to do?))))

Reply:
just remove `aspect_ration_scale: greater` :)